Jellyfin.ApiClient/WebSocket/WebSocketMessageType.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Jellyfin.ApiClient.WebSocket
+{
+    /// <summary>
+    /// Enum WebSocketMessageType
+    /// </summary>
+    public enum WebSocketMessageType
+    {
+        /// <summary>
+        /// The text
+        /// </summary>
+        Text,
+        /// <summary>
+        /// The binary
+        /// </summary>
+        Binary,
+        /// <summary>
+        /// The close
+        /// </summary>
+        Close
```

Comment:
The comments here are pretty useless so I would remove them.

Reply:
Done in 3641a54. Sorry, copy/paste from old code.